src/odemis/driver/smaract.py

```diff
@@ -4043,8 +4282,11 @@ def SA_SI_WaitForEvent(self, handle, event, timeout):
         ev = _deref(event, SA_SI_Event)
```

Comment:
Note that this function actually doesn't really behaves as the real C library function, as it doesn't block. That said, for the current behaviour that might be enough. Maybe add a tiny sleep(0.01) in case there no cancellation to make sure we don't use the CPU at full throttle during simulated calibration.